supybot_fedora/plugin.py

```diff
@@ -52,6 +52,8 @@
 from fedora.client import AppError, AuthError
 from fedora.client.fas2 import AccountSystem
 
+import fasjson_client
+
```

Comment:
Not sure if this has to be in this PR, but at some point the code probably needs to cope with `fedora.client.fas2` going away, but let's leave that for later.